content/hi/docs/concepts/containers/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: कंटेनर्स (Containers)
+weight: 40
+description: एक एप्लिकेशन के साथ उसकी रनटाइम डिपेंडेंसी को पैकेजिंग करने के लिए प्रौद्योगिकी।
+content_type: concept
+---
+
+<!-- overview -->
+
+आप जिस भी कंटेनर को चलाते हैं, वह दोहराया जा सकता है; समेतता से निर्भरताओं के शामिल होने से अर्थ है कि जहां आप भी उसे चलाते हैं, आपको उसी व्यवहार का पता चलता है।
+
+कंटेनर अनुप्रयोगों को मूल अधोसंरचना से अलग करते हैं। इससे विभिन्न क्लाउड या ओएस पर्यावरणों में परिनियोजन आसान हो जाता है।
```

Comment:
`कंटेनर एप्लिकेशनों को उनके आधारभूत होस्ट इंफ्रास्ट्रक्चर से अलग कर देते हैं।
`
For first sentence Simple hindi word will be best for the user to understand. As these words in hindi is hard itself for first go अधोसंरचना ,

instead of `ओएस  `use `OS `itself .
instead of `पर्यावरणों  `use `एनवायरनमेंट `